### ISSUE TITLE
Profiles

### ISSUE THREAD

**Issue by dhaos21** [xbmc/chorus2#244]
Is there any current way to change profiles from within the chorus ui?

**Comment by jez500** [comment]
No, but it looks like the api supports it `#lab/api-browser/Profiles.LoadProfile` so maybe in the future?

**Comment by marapavelka** [comment]
It would be great to add this feature.
